qt/python/mantidqt/widgets/codeeditor/test/test_multifileinterpreter_view.py

```diff
@@ -56,7 +61,7 @@ def test_editor_widget_deletes_find_replace_dialog(self):
         self.assert_number_of_widgets_matching("Embedded", 0)
 
     def test_editor_widget_doesnt_create_find_replace_unless_requested(self):
-        widget = MultiPythonFileInterpreter()
+        widget = MultiPythonFileInterpreter(None)
```

Comment:
This None isn't necessary. It is a default in the `__init__` I believe this may be left over from the CONF being passed to constructors?

Reply:
Seems like it is leftover, good catch!